docker/setup_configuration/data.yaml

```
@@ -0,0 +1,45 @@
+oidc_db_config_enable: true
+oidc_db_config_admin_auth:
+  items:
+    - identifier: admin-oidc
+      oidc_rp_client_id: client-id
+      oidc_rp_client_secret: secret
+      endpoint_config:
+        oidc_op_authorization_endpoint: https://example.com/realms/test/protocol/openid-connect/auth
```

You could set these to localhost:8080, that way it works with the docker-compose from `mozilla-django-oidc-db`
```suggestion
        oidc_op_authorization_endpoint: https://localhost:8080/realms/test/protocol/openid-connect/auth
```